ironfish/src/node.ts

```diff
@@ -214,7 +215,7 @@ export class FullNode {
     dataDir = dataDir || DEFAULT_DATA_DIR
 
     if (!config) {
-      config = new Config(files, dataDir)
+      config = WalletConfig.NewConfig(files, dataDir)
```

Comment:
This is demonstrating that an extended config could be passed into the SDK components.